CMakeLists.txt

```diff
@@ -34,42 +26,25 @@ if(POLICY CMP0012)
   cmake_policy(SET CMP0012 NEW)
 endif()
 
-find_package(OpenMP)
+if(OPENMP)
+  find_package(OpenMP)
```

Comment:
this should be `REQUIRED` if the user wishes to turn this option. Additionally COMPONENTS should be specified.